packages/gatsby/cache-dir/head/head-export-handler-for-browser.js

```diff
@@ -15,7 +15,7 @@ import {
 const hiddenRoot = document.createElement(`div`)
 
 const removePrevHeadElements = () => {
-  const prevHeadNodes = [...document.querySelectorAll(`[data-gatsby-head]`)]
+  const prevHeadNodes = document.querySelectorAll(`[data-gatsby-head]`)
   prevHeadNodes.forEach(e => e.remove())
```

Comment:
This will break in older browser versions where nodeList does not contain forEach. You'll have to figure out the issue differently. What exactly is the issue with [...]?

Reply:
> What exactly is the issue with [...]?

https://github.com/gatsbyjs/gatsby/discussions/36247#discussioncomment-3325711

Good catch 👍 I'd do a `for...of` instead

Reply:
What happens if you change it into Array.from ?

Reply:
`Array.from` works fine — turns out we didn't even need to create new arrays in the first place